core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -574,14 +575,19 @@ private ChannelFuture doStart(ServerPort port) {
         }
 
         private void setupServerMetrics() {
-            final MeterRegistry meterRegistry = config().meterRegistry();
+            final MeterRegistry meterRegistry = config.meterRegistry();
+            final ServerMetrics serverMetrics = config.serverMetrics();
             final GracefulShutdownSupport gracefulShutdownSupport = this.gracefulShutdownSupport;
             assert gracefulShutdownSupport != null;
 
             meterRegistry.gauge("armeria.server.pending.responses", gracefulShutdownSupport,
                                 GracefulShutdownSupport::pendingResponses);
```

Comment:
@seongbeenkim is going to add the metric to the `ServerMetrics` in the follow-up PR.

Reply:
Question) Could `"armeria.server.pending.responses"` be deprecated?
I guess `ServerMetric. activeRequests()` == `GracefulShutdownSupport.pendingResponses()`.

Reply:
`pendingResponses` does not include the transient service requests, meanwhile `activeRequests` include them.
If we make `activeRequests` do not contain the transient service requests, we can deprecate `pendingResponses`. But I wasn't sure.

Reply:
It is hard to notice from the metric name that the difference between the two metrics is the existence of transient services.
We may add an API and a new tag to explicitly expose the number of transient service requests in the f/u work.

Reply:
Yeah, that's a good suggestion. 👍